terraform/stacks/cognito_aai/app_orcaui.tf

```diff
@@ -44,7 +44,16 @@ resource "aws_cognito_user_pool_client" "orcaui_page_app_client" {
   allowed_oauth_flows_user_pool_client = true
   allowed_oauth_scopes                 = ["email", "openid", "profile", "aws.cognito.signin.user.admin"]
 
-  id_token_validity = 24
+
```

Comment:
This isn't relevant to this PR, but it complains if the validity units/default don't match up.